test/cbmc/proofs/ARP/ARPAgeCache/ARPAgeCache_harness.c

```diff
@@ -25,5 +38,43 @@ NetworkBufferDescriptor_t * pxGetNetworkBufferWithDescriptor( size_t xRequestedS
 
 void harness()
 {
+    NetworkInterface_t xNetworkInterface1, xNetworkInterface2;
+    NetworkEndPoint_t xEndPoint1, xEndPoint2;
+
+    const uint8_t ucIPAddress2[ 4 ];
+    const uint8_t ucNetMask2[ 4 ];
+    const uint8_t ucGatewayAddress2[ 4 ];
+    const uint8_t ucDNSServerAddress2[ 4 ];
+    const uint8_t ucMACAddress[ 6 ];
+
+    /* Assume that the list of interfaces/endpoints is not initialized. */
+    __CPROVER_assume( pxNetworkInterfaces == NULL );
+    __CPROVER_assume( pxNetworkEndPoints == NULL );
```

Comment:
Are these two variables even defined?

Reply:
They are defined in one of the included files. They are defined as NULL there. Just copied the assumes here as well :)